docs/arch/adr-38.md

```diff
@@ -0,0 +1,30 @@
+# ADR 37: Engaging Crowds: Next/previous subjects
```

Comment:
You're not fooling me, ADR 38!

Reply:
Fixed, and rebased to the latest master.